tests/entrypoints/llm/test_accuracy.py

```diff
@@ -42,6 +42,10 @@ def run_test(more_args=None):
             ), f"Expected: {EXPECTED_VALUE} |  Measured: {measured_value}"
 
 
+# TODO: [AlexM] Fix it with new CI/CD tests
+TPU_TP_TEST_STR = ""  #"tensor_parallel_size=4"
```

Comment:
What hardware do we have for TPU ci runner?

Reply:
We are going to add v5e-4 this week